src/oas.md

```diff
@@ -3402,13 +3402,13 @@ To ensure that an ambiguous value (e.g. `"foo"`) is treated as a relative URI re
 Mapping keys MUST be string values, but tooling MAY convert response values to strings for comparison.
 However, the exact nature of such conversions are implementation-defined.
 
-##### Optional `discriminator` property
+##### Optional discriminating property
 
-When the `discriminator` property is defined as optional, the `discriminator` field must include a `default` field that specifies a schema that is expected to validate the structure of the model when the `discriminator` property is not present in the payload. This allows the schema to still be validated correctly even if the discriminator property is missing.
+When the discriminating property is defined as optional, the `discriminator` field must include a `default` field that specifies a schema that is expected to validate the structure of the model when the discriminating property is not present in the payload. This allows the schema to still be validated correctly even if the discriminator property is missing.
```

Comment:
```suggestion
When the discriminating property is defined as optional, the `discriminator` schema must include a `default` field that specifies a schema that is expected to validate the structure of the model when the discriminating property is not present in the payload. This allows the schema to still be validated correctly even if the discriminator property is missing.
```

Reply:
I don't think changing "field" to "schema" is right here. Can you explain why you have suggesting this change?

Reply:
the `discriminator` keyword defines a schema. It's not a standalone field. It requires the `propertyName` keyword per the spec.

```yaml
discriminator:
  propertyName: thing
  default: Dog # < your new keyword
```
vs

```yaml
discriminator: ??
```

Reply:
there were a few locations where this wasn't updated.  They are in the `Outdated` conversations below. Is that an oversight or intentional?

Reply:
I can change `discriminator` field to `discriminator` keyword. Would that address your concerns?

Reply:
I feel like OAS fields should be fields even in the Schema Object.  Sometimes when talking about the Schema Object as one of many Objects I just use "fields" because the context is clearly OAS.  I use "keyword" when the more important context it JSON Schema.  Although I have no stronger argument for this usage.  Perhaps @ralfhandl has an opinion here?

Reply:
the reason I raised this is because the discriminator field is not actually a thing. it's a schema.  You can't use the field by itself, as shown in my example.  Referring to it as a field is not correct, imo.

Reply:
@jeremyfiel I don't know what you mean when you say "the discriminator field is not actually a thing". `discriminator` is a ["fixed field" defined by OpenAPI](https://spec.openapis.org/oas/v3.0.4.html#fixed-fields-20), so by [our established terminology](https://github.com/OAI/OpenAPI-Specification/blob/v3.2-dev/CONTRIBUTING.md#use-of-keyword-field-property-and-attribute), we refer to it as a "field".

Reply:
I understand the field is described by OAS, but the field itself is a schema. It's never a standalone field such as additionalProperties: false or additionalProperties: {} where it can be both. 

In this reference language, I believe the discriminator field should be referenced as a schema because you are referencing the behavior of the field. The use of _field_ is correct when referencing the `defaultMapping` requirement.

> When the discriminating property is defined as optional, the `discriminator` schema must include a `defaultMapping` field that specifies a schema

Reply:
Okay ... I think I see a way to resolve this. What we are talking about here is the "Discriminator Object" -- the value of the `discriminator` field. So we could use "Discriminator Object" rather than "`discriminator` field`" in the places you have flagged. I think this would be consistent with language/terminology elsewhere in the spec. Would this work for you @jeremyfiel ?

Reply:
Sounds good! 👍